src/commands/infra/scaffold.ts

```diff
@@ -60,8 +63,53 @@ export const validateVariablesTf = async (
       );
       return false;
     }
+    logger.info(
+      `Terraform variables.tf file found. Attempting to generate definition JSON/HCL file...`
+    );
   } catch (_) {
-    logger.error(`Unable to Validate Infra Init.`);
+    logger.error(`Unable to validate Terraform variables.tf.`);
+    return false;
+  }
+  return true;
+};
+
+/**
+ * Rename any .tfvars file by appending ".backup"
+ *
+ * @param dir path to template directory
+ */
+export const renameTfvars = async (dir: string): Promise<void> => {
```

Comment:
Rather than copying then renaming TFVar files, why not just avoid copying them in the first place?

Reply:
fs library has a method that allows to copy an entire directly without having to write logic that involves iteration. I feel that it is best to include the tfvars file as part of the copy process, instead of removing or excluding it because it gives users the option to toggle between using Terraform or Terragrunt if that becomes necessary. In the case, where a user wants to use Terraform, they would just need to rename/remove the tfvars file. Do you think we should not provide this option?

Reply:
I think the `tfvar.backup` is a good approach. We should store it just in case the user wishes to pivot from leveraging terragrunt and needs to debug directly with terraform.